frontend/src/modules/page_foundation/main.js

```diff
@@ -1 +1,46 @@
 import './main.scss';
+document.getElementById("contributor").addEventListener('click',function ()
```

Comment:
- This will fail to correctly initialize in some cases. Please, wrap this code in DOMContentLoaded event handler.
- Please use quotes consistent with the rest of the codebase - e.g. single quotes.
- Please terminate all lines with semicolon `;`
- Replace `var` with `const` where appropriate.